WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java

```diff
@@ -993,6 +749,32 @@ public void onInputUpdated(String input) {
         dialog.show(getFragmentManager(), null);
     }
 
+    private void showLocationPicker() {
+        if (!isAdded()) {
+            return;
+        }
+        PlacePicker.IntentBuilder builder = new PlacePicker.IntentBuilder();
+        // Pre-pick the previous selected location if any
+        LatLng latLng = null;
+        if (mPostLocation != null) {
```

Comment:
This again is good for now, but we should get rid of the `mPostLocation` completely and rely on the `mPost` object since we'll be updating it immediately after a change. This is just a reference to the intended change.